src/components/viesti/viestit.svelte

```diff
@@ -38,8 +38,7 @@
   )($querystring);
   $: pageCount = Math.ceil(R.divide(ketjutCount, pageSize));
 
-  const nextPageCallback = nextPage =>
-    push(`#/viesti/all?page=${nextPage}`);
+  const nextPageCallback = nextPage => push(`#/viesti/all?page=${nextPage}`);
```

Comment:
This looks like it is is simply a reformat with no actual change in the program. Would be good to reformat in a separate commit, so that it is easy to ignore these changes while reviewing the changes that need attention.